client/src/pages/Search.js

```diff
@@ -521,17 +521,49 @@ export const Tasks = ({
   }
 
   const tasks = data ? data.taskList.list : []
+
+  return (
+    <TasksPagination
+      items={tasks}
+      pageNum={pageNum}
+      pageSize={taskQuery.pageSize}
+      totalCount={totalCount}
+      setPage={setPage}
+    />
+  )
+
+  function setPage(pageNum) {
+    setPagination(paginationKey, pageNum)
+    setPageNum(pageNum)
+  }
+}
+
+Tasks.propTypes = {
+  pageDispatchers: PageDispatchersPropType,
+  queryParams: PropTypes.object,
+  setTotalCount: PropTypes.func,
+  paginationKey: PropTypes.string.isRequired,
+  pagination: PropTypes.object.isRequired,
+  setPagination: PropTypes.func.isRequired
+}
+
+export const TasksPagination = ({
```

Comment:
You should remove the last changes to this file that you made in commit https://github.com/NCI-Agency/anet/commit/441b0da72cc40c4006f6fda61c76ba29cf02bbb1.

I have pushed a  better fix for this (see https://github.com/NCI-Agency/anet/commit/6cdb30982da67672ed5c8c1ef1336fb445be5ae3), which is the way we deal with it in the rest of ANET.

Reply:
fixed